src/main/java/salesforce/ui/pages/product/ProductsPage.java

```diff
@@ -0,0 +1,47 @@
+package salesforce.ui.pages.product;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.BasePage;
+
+/**
+ * This class returns an instance of ProductsPage.
+ */
+public class ProductsPage extends BasePage {
+
+    @FindBy(css = ".forceActionLink > div")
+    private WebElement newProductButton;
+
+    @FindBy(xpath = "//div/a[contains(@class, 'rowActionsPlaceHolder')]")
+    private WebElement showActionsButton;
+
+    /**
+     * Waits for the page to load.
+     */
+    @Override
+    protected void waitForPageToLoad() {
+        getWait().until(ExpectedConditions.visibilityOf(newProductButton));
+    }
+
+    /**
+     * Logs in successfully.
```

Comment:
The method says that it logs in succesfully. the method name says click on new product. It would seem that when clicking on new product you log in.

Reply:
Thanks. I have just fixed that